azure-pipelines.yml

```diff
@@ -287,7 +287,7 @@ stages:
               PYTEST="docker run -e GE_CLOUD_ORGANIZATION_ID=$(GE_CLOUD_ORGANIZATION_ID) "
               PYTEST+="-e GE_CLOUD_ACCESS_TOKEN=$(GE_CLOUD_ACCESS_TOKEN) -e GE_CLOUD_BASE_URL=$(GE_CLOUD_BASE_URL) "
               PYTEST+="greatexpectations/test:${DOCKER_TAG} "
-              PYTEST+="pytest tests --cloud -m \"cloud and e2e\""
+              PYTEST+="pytest --random-order --cloud -m \"cloud and e2e\""
```

Comment:
Does it look for a `tests` directory if you don't tell it to?

Reply:
yes.